picfixPy/test/test_vibrance.py

```diff
@@ -9,15 +9,15 @@
 import numpy as np
 import pytest
 import skimage.io
-from picfixPy import vibrance
+from picfixPy.vibrance import vibrance
```

Comment:
thanks for the update! I'll continue the test once this has been merged.

Reply:
A pleasure!